doc/tutorials/content/dinast_grabber.rst

```diff
@@ -208,4 +208,4 @@ Conclusions
 With this new grabber a new kind of short-range sensors are available through the PCL Grabber interface. 
 It is now a breeze to connect and obtain data from Dinast devices as you do with the rest of devices supported at PCL.
 
-If you have any development suggestions on these or new devices you can contact us through *pcl-developers@pointclouds.org*.
\ No newline at end of file
+If you have any development suggestions on these or new devices you can contact us through *pcl-developers@pointclouds.org*.
```

Comment:
I can't figure out what was changed here. Github isn't marking anything and the content seems similar. Do you still remember?

Reply:
not sure what's happening to the usual website icon :man_shrugging:. thanks for the clarification